### PR TITLE
add metrics docs

### PR DESCRIPTION
**Pull request by clouserw**
This is a first draft.  Ready for people to comment on it, but not ready to land.

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+for details.  That means the "how" part of this document might change, but the
+  "what" part should be pretty accurate.
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people want to use this?
+
+What is the overall usage of FirefoxNoMore494s?  **This is the standard Daily Active User
```
Comment (pdehaan) [inline docs/metrics.md]: **Typo:** FirefoxNoMore404s (not 494)

File: docs/metrics.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+* 'none': will be 'none' if there isn't an archive available
+
+Here's an example of the `payload` portion of a Test Pilot telemetry ping:
+
+```js
+{
+  "test": "Wayback Machine",         // The name of the add-on
+  "agent": "User Agent String",
+  "payload": {
+    "action": "viewed"              // See full list above
```
Comment (pdehaan) [inline docs/metrics.md]: **Nit:** missing a trailing comma here (and have a bonus trailing comma below on L70).

File: docs/metrics.md (hunk 1 above)
Comment (wresuolc) [inline docs/metrics.md]: 💯

**Comment by clouserw** [comment]
https://github.com/mozilla/testpilot/pull/1008 is the code that (we think) will allow a webextension to use telemetry.  Once that lands we can start building

**Comment by lmorchard** [comment]
Found a cleaner / simpler way here https://github.com/mozilla/testpilot/pull/1022

Example usage on the webextension side looks like this https://github.com/mozilla/testpilot/pull/1022/commits/0f38165f03b07ba0f22a7d09a2d550238ef2f0dd#diff-471217898021b66d8e609e881600b0f3R1

**Pull request merged**